scripts/change_url

```diff
@@ -116,10 +34,10 @@ ynh_script_progression --message="Updating a configuration file..." --weight=1
 
 domain=$new_domain
 path=$new_path
-ynh_add_config --template="../conf/config.ini" --destination="$final_path/config.ini"
+ynh_add_config --template="../conf/config.ini" --destination="$install_dir/config.ini"
```

Comment:
Will that keep the previous values (except the base URL) of the config.ini file? In particular the salt which should not change?

Reply:
Previous values should be kept (to be tested of course)